script/tool/lib/src/update_excerpts_command.dart

```diff
@@ -142,10 +142,15 @@ class UpdateExcerptsCommand extends PackageLoopingCommand {
                   case '.kt':
                     language = 'kotlin';
                     break;
-                  case '.cc':
```

Comment:
Why was this removed?

Reply:
Shoot, there was a problem with my test case for `.cc` files. My mistake.